README.md

```diff
@@ -25,3 +37,41 @@ const np = new NowPlaying(Providers.SPOTIFY, {
   },
 });
 ```
+
+### Storage Kinds
+
+- **INMEMORY**: Stores data in memory. This is the default option and suitable to reduce overhead.
+
+### Providers
+
+#### Spotify
+
+You need two things.
+
+1. Spotify Client ID and Secret
+2. Spotify Refresh Token
+
+To get your Spotify Client ID and sercet, follow these steps:
+
+1. Log in to your [Spotify Dashboard](https://developer.spotify.com/dashboard/applications).
+2. Create a new app by clicking on the "Create an App" button. You can name your app whatever you like.
+3. After creating the app, you will see the details page for your app. Here, you can find your client ID and client secret.
+
+You'll need these credentials to generate a refresh token for Spotify.
+
+Spotify's access token expires every hour, so you will need a refresh token to obtain a new one automatically. To get your refresh token, refer to the following articles for a detailed guide:
+
+- [How to create a Spotify refresh token](https://benwiz.com/blog/create-spotify-refresh-token/)
+- [Connecting your React Vite.js app to Spotify's API](https://reine.hashnode.dev/how-to-connect-your-react-vitejs-app-to-spotifys-ap)
```

Comment:
Yeah, we need a blog post specificallly on how you added this to the portfolio.

I think this should be written by you since you're the brain child of this project and it should be in your microsite.

Reply:
Yeah, that makes sense.

Let me get to work then.